tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/LdmlConverter.java

```diff
@@ -344,7 +273,20 @@ private void processAndSplitLocaleFiles(
             if (!availableIds.contains(id)) {
                 continue;
             }
-            IcuData icuData = icuFn.apply(id);
+
+            IcuData icuData = new IcuData(id, true);
```

Comment:
Now we write a single IcuData instance and load specials data only once.
All mappers are executed and then we split results according to the resource bundle.